packages/ai/src/mcp.ts

```diff
@@ -0,0 +1,17 @@
+import { Function } from './function';
```

Comment:
I see why you need to do this. To me making changes in the core packages for these kinds of things (external integrations that come and go often) is a smell that the AI package isn't flexible enough and needs more plugin/extension capabilities.

What I mean by that is that with teams AI lots of trendy features came and went, and every time we added to the core package we incurred dept because the features rarely have similar schemas. The same applies to the Apps package which is why I implemented the plugins abstraction. I'd like us to find some way to do something similar here, so we can build integrations as lego pieces on top of the core

Reply:
for example, last year all the hype was around the agent protocol https://agentprotocol.ai/

Reply:
Added AI Plugins